### PR TITLE
separate wrapper function

### PR DESCRIPTION
**Pull request by ngreenwald**
This provides a way for us to modify the post-processing params that are sent to the consumer.

Let me know if you think it would be better to have this somewhere else before I start moving forward

### CODE REVIEW

File: redis_consumer/processing.py [hunk 1]
```diff
@@ -53,3 +53,34 @@
...
+        np.ndarray: labeled image
+    """
+
+    if whole_cell_kwargs is None:
+        whole_cell_kwargs = {}
+        whole_cell_kwargs['radius'] = 5
+
+    if nuclear_kwargs is None:
+        nuclear_kwargs = {}
+        nuclear_kwargs['radius'] = 5
```
Comment (willgraf) [inline redis_consumer/processing.py:80]: Is `radius` the only one we want to default?

**Review by willgraf — CHANGES_REQUESTED** [review]
I think the consumer itself is still using `multiplex_postprocess` instead of the new value.

**Comment by ngreenwald** [comment]
It's the only one that's different from the defaults in deepcell_toolbox.
Should we move all of the defaults to here?

**Comment by ngreenwald** [comment]
Actually nevermind, I'm going to update them with the newest
post-processing values, so I'll update all of them, and set the defaults
here. Will make things easier to update moving forward.

**Comment by willgraf** [comment]
Yeah I think it makes sense to add all the values here so they're easier to tweak moving forward.

**Comment by ngreenwald** [comment]
Okay, I updated it so there are defaults for all of them. 

Currently, the consumer post-processing function doesn't take additional arguments. This doesn't matter right now, since there's no way for users to control any of the post-processing parameters. Eventually, we might want to add that option to the ImageJ app or deepcell.org. When then happens, we'd need to add a way for the base post-processing class to handle additional arguments. 

However, since right now this doesn't exist, I set it up so that the 'defaults' are always used.

**Review by willgraf — APPROVED** [review]
This looks good, but I'd like to test the consumers out before we merge it.

**Comment by ngreenwald** [comment]
Yup, deepcell.org and ImageJ both gave good looking results

**Pull request merged**